src/System.Net.HttpListener/tests/HttpListenerAuthenticationTests.cs

```diff
@@ -414,8 +433,9 @@ public void Realm_SetDisposed_ThrowsObjectDisposedException()
 
             // The server task will hang forever if it is not cancelled.
             var tokenSource = new CancellationTokenSource();
-            Task<HttpListenerContext> serverTask = Task.Run(() => _listener.GetContext(), tokenSource.Token);
 
+            Task<HttpListenerContext> serverTask = Task.Run(() => _listener.GetContext(), tokenSource.Token);
```

Comment:
It looks like you're not actually changing anything here, but the existing code is a bit bogus... this cancellation token is essentially a nop given how it's used. Can be addressed subsequently.